### ISSUE TITLE
msgfmt: error while opening "/var/tmp/portage/app-portage/eix-0.33.5_alpha20180804/work/eix-ae6d36ef79c260d620067ddc7b4646128eda8991/po/en@quot.po" for reading: No such file or directory

### ISSUE THREAD

**Issue by l29ah** [vaeth/eix#54]
```
Running custom install script '/usr/bin/python3.5 /usr/lib64/python-exec/python3.5/meson --internal gettext install --subdir=po --localedir=share/locale --pkgname=eix'
msgfmt: error while opening "/var/tmp/portage/app-portage/eix-0.33.5_alpha20180804/work/eix-ae6d36ef79c260d620067ddc7b4646128eda8991/po/en@quot.po" for reading: No such file or directory
Traceback (most recent call last):
  File "/usr/lib64/python-exec/python3.5/meson", line 29, in <module>
    sys.exit(mesonmain.main())
  File "/usr/lib64/python3.5/site-packages/mesonbuild/mesonmain.py", line 379, in main
    return run(sys.argv[1:], launcher)
  File "/usr/lib64/python3.5/site-packages/mesonbuild/mesonmain.py", line 316, in run
    sys.exit(run_script_command(args[1:]))
  File "/usr/lib64/python3.5/site-packages/mesonbuild/mesonmain.py", line 232, in run_script_command
    return cmdfunc(cmdargs)
  File "/usr/lib64/python3.5/site-packages/mesonbuild/scripts/gettext.py", line 118, in run
    if gen_gmo(src_sub, bld_sub, langs) != 0:
  File "/usr/lib64/python3.5/site-packages/mesonbuild/scripts/gettext.py", line 66, in gen_gmo
    '-o', os.path.join(bld_sub, l + '.gmo')])
  File "/usr/lib64/python3.5/subprocess.py", line 271, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['msgfmt', '/var/tmp/portage/app-portage/eix-0.33.5_alpha20180804/work/eix-ae6d36ef79c260d620067ddc7b4646128eda8991/po/en@quot.po', '-o', '/var/tmp/portage/app-portage/eix-0.33.5_alpha20180804/work/eix-0.33.5_alpha20180804-build/po/en@quot.gmo']' returned non-zero exit status 1
FAILED: meson-install 
/usr/bin/python3.5 /usr/lib64/python-exec/python3.5/meson install --no-rebuild
ninja: build stopped: subcommand failed.
```

**Comment by vaeth** [comment]
Thanks for the report.
This should be fixed in commit 4ac7d32a0daf3c7e67c2d1b3eeb87eedf0d5a19b.
A corresponding update of the alpha version is in the mv overlay.

**Comment by l29ah** [comment]
Thanks!
